src/checkbox.js

```diff
@@ -0,0 +1,47 @@
+const tasksArray = [
+  {
+    description: 'Thinking of a fat joint',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Rolling a fat joint',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Smoking of a fat joint',
+    completed: false,
+    index: 2,
+  },
+];
+
+const tasks = document.querySelector('.tasks');
+
+function render() {
+  for (let i = 0; i < tasksArray.length; i += 1) {
```

Comment:
- [ ] The UI state is not in sync with the localStorage state. After a page reload the completed tasks are reset to `false` and this is not what is expected. Let's use the localStorage `tasksArray` value in this function to fix that issue.

![Screenshot from 2021-11-03 18-54-09](https://user-images.githubusercontent.com/40503704/140165506-a9443191-2d8e-4342-87f8-425f82e1feab.png) 
![Screenshot from 2021-11-03 18-54-21](https://user-images.githubusercontent.com/40503704/140165536-61725430-2cad-46fb-ad0c-71cf625ab877.png)

Reply:
This issue is well resolved! ✔️